chaos_genius/core/rca/rca_utils/api_utils.py

```diff
@@ -38,6 +39,16 @@ def kpi_aggregation(kpi_id, timeline="last_30_days"):
             .first()
         )
 
+        rca_end_date = data_point.end_date
+
+        anomaly_data_point = AnomalyDataOutput.query.filter(
+            (AnomalyDataOutput.kpi_id == kpi_id)
+            & (AnomalyDataOutput.anomaly_type == "overall")
+            & (AnomalyDataOutput.is_anomaly != 0)
+            & (AnomalyDataOutput.data_datetime <= rca_end_date + timedelta(days=7))
```

Comment:
Why is this +7 days from end date? Shouldn't it be for past 7 days (end_date - 7 days)?

Reply:
Fixed